.github/workflows/ci-lint.yaml

```diff
@@ -9,6 +9,13 @@ jobs:
         uses: golangci/golangci-lint-action@v2
         with:
           version: v1.31
+  bash-lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Run Shellcheck
+        uses: azohra/shell-linter@latest
+        run: |
+          shellcheck -x ./scripts/*.sh
```

Comment:
Trying this out; the instructions online show me how to do a few things, but not this exactly. Will watch CI and see how it goes.